cypress/e2e/admin/administrateCtlTools.feature

```diff
@@ -0,0 +1,21 @@
+@release
```

Comment:
please comment that out or delete the line as long as QA did not approve this PR

Reply:
done